staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -40,9 +40,10 @@ var (
 	// deprecatedRequestLatency is deprecated, please use requestLatency.
 	deprecatedRequestLatency = k8smetrics.NewHistogramVec(
 		&k8smetrics.HistogramOpts{
-			Name:    "rest_client_request_latency_seconds",
-			Help:    "(Deprecated) Request latency in seconds. Broken down by verb and URL.",
-			Buckets: k8smetrics.ExponentialBuckets(0.001, 2, 10),
+			Name:              "rest_client_request_latency_seconds",
+			Help:              "Request latency in seconds. Broken down by verb and URL.",
+			Buckets:           k8smetrics.ExponentialBuckets(0.001, 2, 10),
+			DeprecatedVersion: "1.16.0",
```

Comment:
These were deprecated in 1.14.

Reply:
I talked with @logicalhan about this. We like to hide them in v1.17 and remove them from code base in v1.18. 

IMO, we can take this as an exercise for our stability framework. If we set `DeprecatedVersion` with `1.14.0` or `1.15.0` ,that will be equal to remove them from codebase.

Reply:
I’m generally not happy to go against what we specified in the KEP, but I think we can make an exception here as it hasn’t really hurt us thus far. We should not be pushing this out any further though.

Reply:
Wow, seems we should do something here, the metrics with `deprecated version` - `current version` > 2, should be hidden, even turn on the flag `--show-hidden-metrics`.
 
https://github.com/kubernetes/kubernetes/blob/bcb171b375df0dd9326bf5d3a23428beea1aeb89/staging/src/k8s.io/component-base/metrics/metric.go#L100-L107

Reply:
> I’m generally not happy to go against what we specified in the KEP, but I think we can make an exception here as it hasn’t really hurt us thus far. 

Fine. If @logicalhan no other comments, i will update deprecated version to 1.14.0.

Reply:
Effectively, it's the same, so I'm okay with annotating the metadata as deprecated since 1.14. I guess the only part about it that is weird is that the stability framework didn't itself exist until 1.16, so there is something temporally confusing about marking them as deprecated since 1.14.

Reply:
I think that’s ok given this is an in progress state.